samples/system-test/spanner.test.js

```diff
@@ -41,277 +41,298 @@ const spanner = new Spanner({
+      labels: {
+        'gcloud-sample-tests': 'true',
+      },
+    });
+
+    await operation.promise();
   });
 
-  await operation.promise();
-});
-
-test.before(async () => {
-  const [instances] = await spanner.getInstances({
-    filter: 'labels.gcloud-sample-tests:true',
-  });
+  before(async () => {
+    const [instances] = await spanner.getInstances({
+      filter: 'labels.gcloud-sample-tests:true',
+    });
 
-  instances.forEach(async instance => {
-    const {operations} = await getOperations(instance.metadata.name);
+    instances.forEach(async instance => {
+      const {operations} = await getOperations(instance.metadata.name);
 
-    operations
-      .filter(operation => {
-        return operation.metadata['@type'].includes('CreateInstance');
-      })
-      .filter(operation => {
-        const yesterday = new Date();
-        yesterday.setHours(-24);
+      operations
+        .filter(operation => {
+          return operation.metadata['@type'].includes('CreateInstance');
+        })
+        .filter(operation => {
+          const yesterday = new Date();
+          yesterday.setHours(-24);
 
-        const instanceCreated = new Date(operation.metadata.startTime);
+          const instanceCreated = new Date(operation.metadata.startTime);
 
-        return instanceCreated < yesterday;
-      })
-      .forEach(async () => await instance.delete());
+          return instanceCreated < yesterday;
+        })
+        .forEach(async () => await instance.delete());
+    });
   });
-});
 
-test.after.always(async () => {
-  const instance = spanner.instance(INSTANCE_ID);
-  const database = instance.database(DATABASE_ID);
-
-  try {
-    await database.delete();
-  } catch (err) {
-    // Ignore error
-  }
-
-  try {
-    await instance.delete();
-  } catch (err) {
-    // Ignore error
-  }
-});
+  afterEach(async () => {
+    const instance = spanner.instance(INSTANCE_ID);
+    const database = instance.database(DATABASE_ID);
+
+    try {
+      await database.delete();
+    } catch (err) {
+      // Ignore error
+    }
+
+    try {
+      await instance.delete();
+    } catch (err) {
+      // Ignore error
+    }
+  });
 
-// create_database
-test.serial(`should create an example database`, async t => {
-  const results = await tools.runAsyncWithIO(
-    `${schemaCmd} createDatabase "${INSTANCE_ID}" "${DATABASE_ID}" ${PROJECT_ID}`,
-    cwd
-  );
-  const output = results.stdout + results.stderr;
-  t.regex(
-    output,
-    new RegExp(`Waiting for operation on ${DATABASE_ID} to complete...`)
-  );
-  t.regex(
-    output,
-    new RegExp(`Created database ${DATABASE_ID} on instance ${INSTANCE_ID}.`)
-  );
-});
+  // create_database
+  it(`should create an example database`, async () => {
+    const results = await tools.runAsyncWithIO(
+      `${schemaCmd} createDatabase "${INSTANCE_ID}" "${DATABASE_ID}" ${PROJECT_ID}`,
+      cwd
+    );
+    const output = results.stdout + results.stderr;
+    assert.deepStrictEqual(
```

Comment:
I'd be surprised if that worked :) I think you need to `assert.strictEquals(output.matches(someRegex), true)`

Reply:
Thank you Justin for correcting me. I did the requested change.